### PR TITLE
US003 View Capability Role

### PR DESCRIPTION
**Pull request by caitlinbulloch**
**View the Capability a Role belongs too:**
- Updated the Job Class to include 'Capability Name'.
- Updated the JobDao Class to include SELECT statement that gathers capabilityName from the database.
- Edited pom.xml file to allow testing dependencies. 
- Created unit testing for the getAllJobs method. 
- Created testing for the DatabaseConnector.

Screenshot 2023-12-14 at 09 23 58

### CODE REVIEW

File: src/main/java/org/kainos/ea/DropwizardApplication.java [hunk 1]
```diff
@@ -30,6 +32,7 @@ protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(DropwizardCon
     public void run(final DropwizardConfiguration configuration,
                     final Environment environment) {
         // TODO: implement application
```
Comment (jkelso662) [inline src/main/java/org/kainos/ea/DropwizardApplication.java]: Now that you have implemented you first controller you can remove this comment. Not a blocker tho :)

File: src/main/java/org/kainos/ea/cli/JobRequest.java
Comment (jkelso662) [inline src/main/java/org/kainos/ea/cli/JobRequest.java:1]: This file I think is a duplication of `Job.java`

File: src/test/java/db/DatabaseConnectorTest.java
Comment (jkelso662) [inline src/test/java/db/DatabaseConnectorTest.java:1]: Fab 👍

File: src/test/java/service/JobServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    Write a unit test for the getJobs method
+
+    When the dao returns a list of employees
+
+    Expect the list of employees to be returned
+
+    This should pass without code changes
+
+     */
```
Comment (jkelso662) [inline src/test/java/service/JobServiceTest.java]: Brilliant to see automated tests!!! 💯  Just a note that you don't need these types of comments - that was just the way Shaun wrote the tasks in the exercises

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Great PR description and a really well rounded PR! Great job. 
Unfortunately there are a few things to call out...

No one from your team has reviewed this PR yet. Remeber to get someone else to review before asking me and Joanne to take a look :) We have reviewed now to speed yous up tho. Could you rename your PR title so that it includes the US part of the ticket number? Sometime integration tools need the name to match a specific format so best just keeping that consistent :) Also your linter is failing with a bunch of style checks that need fixed. There are a couple of other comments, only major one is the duplicated Job file.

That sounds like a lot but tbh it's just wee niggly things. The majority of the code is actually great :)

**Review by Cormac-F — APPROVED** [review]
LGTM!

**Review by lullabelle — APPROVED** [review]
approved but maybe take the comments out of your test file. The name of the test should explain what its doing

**Pull request merged**